docs/source/azure.rst

```diff
@@ -24,12 +24,29 @@ definition, the following options are available.
 | role             | true       | string        | N/A               |                 |
 +------------------+------------+---------------+-------------------+-----------------+
 | vm_name          | true       | string        | name              |It can't include |
-|                  |            |               |                   | _ and other     |
+|                  |            |               |                   | '_' and other   |
 |                  |            |               |                   |special char     |
 +------------------+------------+---------------+-------------------+-----------------+
-| image            | true       | string        | image             |                 |
+| private_image    | false      | string        | image             |This para takes  |
```

Comment:
is 'para' means 'parameter'?  the comment is quite obvious, I think you can mention the priority of private_image here.